pyvista/utilities/misc.py

```diff
@@ -35,6 +35,38 @@ def _check_range(value, rng, parm_name):
         )
 
 
+def try_imageio_imread(filename):
```

Comment:
Will this function end up in the pyvista namesapce?  Do we want it to?  It is mostly an internal PyVista function that checks for importability, so it seems not that useful for downstream users.  If this currently ending up in the namespace, then I'd argue that we make this private instead.

Reply:
Shoot. I'll fix this.